src/Maestro/Maestro.Data/DarcRemoteFactory.cs

```diff
@@ -9,15 +9,14 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Maestro.AzureDevOps;
-using Maestro.Data;
 using Microsoft.Dotnet.GitHub.Authentication;
 using Microsoft.DotNet.DarcLib;
 using Microsoft.DotNet.ServiceFabric.ServiceHost;
 using Microsoft.DotNet.Services.Utility;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 
-namespace SubscriptionActorService
+namespace Maestro.Data
```

Comment:
Why are these types being moved to "Data"? They don't have anything to do with the database layer, so this doesn't seem like an appropriate place for them.